features_and_tools/bulkdataexport.rst

```diff
@@ -233,5 +237,14 @@ ISystemBulkDataExportRepository
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 Used when performing a System level export. It should retrieve the snapshot, use this to obtain the relevant data from the proprietary datastore and transform this to FHIR resources.
 
+.. note::
 
+  The interfaces below can be found in Vonk.Core version 5.1.0 and higher.
   
+IPatientBulkDataWithPatientsFilterExportRepository
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Optional addition. Used when performing a Patient level export with the 'patient' parameter in the request. It should filter the patients from the snapshot based on the references provided.
```

Comment:
How about adding a link to the BDE OperationDefinition?

Reply:
done